examples/aws-nodejs/public/index.html

```diff
@@ -13,11 +13,29 @@ <h2>AWS S3 multipart</h2>
     <div id="aws-multipart"></div>
     <script type="module">
       import { Uppy, Dashboard, AwsS3Multipart, AwsS3 } from "https://releases.transloadit.com/uppy/v3.10.0/uppy.min.mjs"
+      function* serializeSubPart(key, value) {
+        if (typeof value !== "object") {
+          yield [key, value];
+          return;
+        }
+        if (Array.isArray(value)) {
+          for (const val of value) {
+            yield* serializeSubPart(`${key}[]`, val);
+          }
+          return;
+        }
+        for (const [subkey, val] of Object.entries(value)) {
+          yield* serializeSubPart(key ? `${key}[${subkey}]` : subkey, val);
+        }
+      }
+      function serialize(data) {
+        return new URLSearchParams(serializeSubPart(null, data));
```

Comment:
```suggestion
        // If you want to avoid preflight requests, use URL-encoded syntax:
        return new URLSearchParams(serializeSubPart(null, data))
        // If you don't care about additional preflight requests, you can also use:
        // return JSON.stringify(data)
        // You'd also have to add `Content-Type` header with value `application/json`.
```